src/chat/infinite.js

```diff
@@ -286,6 +346,26 @@ export function infiniteMessageArea (dom, kb, chatChannel, options) {
         'keydown',
         async function (e) {
           // User preference?
+          debug.log('key code ' + e.keyCode)
+          if (e.shiftKey) {
+            if (e.keyCode === 50) {
+              debug.log('testing @ key')
```

Comment:
You are checking for Shift+2, wich is a `"` on a german keyboard, not an `@`. I think you need to check if the @ character is typed, regardless of the keycode that produced it.